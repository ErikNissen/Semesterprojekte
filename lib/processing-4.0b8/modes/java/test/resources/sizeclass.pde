version https://git-lfs.github.com/spec/v1
oid sha256:3c7d5419179904c1a1bc3ae4b7787a19c182cd7d89f49f9c526bd1675977d9d1
size 339
