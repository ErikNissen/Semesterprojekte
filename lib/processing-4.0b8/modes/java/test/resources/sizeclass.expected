version https://git-lfs.github.com/spec/v1
oid sha256:040e262f0c4a15dc23840bca9e2956b2c80708d4ef64579abe59405e66e9fb01
size 1098
